common/arch/win32/rbaudio.cpp

```diff
@@ -372,7 +372,7 @@ void RBACheckFinishedHook()
         if (thisFrames == lastFrames)
             checkValue -= 64;
 
-        if (playEnd >= 0 && lastFrames >= checkValue)
+        if (redbook_finished_hook && playEnd >= 0 && lastFrames >= checkValue)
```

Comment:
Please squash this into the commit that added the call.

Reply:
Wouldn't it make more sense to just squash all commits thus far?